StarkInfra/StarkInfra/CreditNote/Transfer.cs

```diff
@@ -59,21 +59,21 @@ public partial class Transfer : Resource
         /// Parameters (required):
         /// <list>
         ///     <item>name [string]: receiver full name. ex: "Anthony Edward Stark"</item>
-        ///     <item>taxId [string]: receiver tax ID (CPF or CNPJ) with or without formatting. ex: "01234567890" or "20.018.183/0001-80"</item>
+        ///     <item>taxID [string]: receiver tax ID (CPF or CNPJ) with or without formatting. ex: "01234567890" or "20.018.183/0001-80"</item>
         ///     <item>bankCode [string]: code of the receiver Infra institution in Brazil. If an BankCode (8 digits) is informed, a PIX transfer will be created, else a TED will be issued. ex: "20018183" or "341"</item>
```

Comment:
`bankCode [string]: code of the receiver Infra institution in Brazil. ex: "20018183"`